typespec/Applications.Core/secretstores.tsp

```diff
@@ -68,6 +68,15 @@ enum SecretStoreDataType {
 
   @doc("Certificate secret data type")
   certificate,
+
+  @doc("basicAuthentication type is used to represent username and password based authentication and the secretstore resource is expected to have the keys 'username' and 'password'.")
+  basicAuthentication,
+
+  @doc("azureWorkloadIdentity type is used to represent registry authentication using azure federated identity and the secretstore resource is expected to have the keys 'clientId' and 'tenantId'.")
```

Comment:
Do we need to call out "registry authentication" here? This resource is/can be referenced in multiple places with use cases being not limited to ACR. I'd also get PMs eyes on these user facing descriptions.

Reply:
cc @vishwahiremat, @Reshrahim . referring to [link](https://github.com/radius-project/design-notes/pull/53/files#diff-12354a738cc93ef8bfee787de4ebb16befdb3c7dccce0df0a0f4e29e523b779dR288)

Reply:
I agree with not specifying `registry` here as it can be referenced elsewhere as well.
```suggestion
  @doc("azureWorkloadIdentity type is used to represent authentication using azure federated identity and the secretstore resource is expected to have the keys 'clientId' and 'tenantId'.")
```

Reply:
```suggestion
  @doc("azureWorkloadIdentity type is used to represent registry authentication using azure workload identity and the secretstore resource is expected to have the keys 'clientId' and 'tenantId'.")
```

Reply:
I think it is references as Azure workload identity and not federated identity in the docs

Reply:
Thanks @Reshrahim. What are your thoughts on explicitly mentioned "registry" authentication? Secret store resource is not limited to supporting management of secrets only for the use case of registries.

Reply:
Yeah sorry din notice the comment above. I agree with your comment of calling it as authentication as the uses cases can be outside of registries

Reply:
Thanks for confirming @Reshrahim.

@vishwahiremat @lakshmimsft - could one of you please update this as per Reshma's feedback?